.github/workflows/test-web.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - run:  |
+      - run: |
           npm install -g corepack@latest
           corepack enable pnpm
 
```

Comment:
These actions are perfect! One thing for future is to enable caching when it becomes a bottleneck https://github.com/pnpm/action-setup?tab=readme-ov-file#use-cache-to-reduce-installation-time